test/unit/test-amp-story-player.js

```diff
@@ -370,5 +370,78 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
         'Story URL not found in the player: https://example.com/story6.html'
       );
     });
+
+    it('navigate forward given a positive number in range', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      win.requestAnimationFrame(() => {
+        const iframes = playerEl.shadowRoot.querySelectorAll('iframe');
+        player.go(2);
+        expect(iframes[1].getAttribute('i-amphtml-iframe-position')).to.eql(-1);
+        expect(iframes[2].getAttribute('i-amphtml-iframe-position')).to.eql(0);
+        expect(iframes[3].getAttribute('i-amphtml-iframe-position')).to.eql(1);
```

Comment:
I don't think there's a 4th iframe. There's maximum 3.

Reply:
Did the test still pass?

Reply:
(See comment below) I think there's an issue with the tests and this code was not actually running

Reply:
We could either return a Promise that resolves after these `expect` statements, or get a `done` parameter from the `it()` second callback parameter, that we call after the `expect` statements.
`it('navigate forward given a positive number in range', async (done) => {`